src/Workspaces/Core/Portable/Formatting/Engine/AbstractFormatEngine.OperationApplier.cs

```diff
@@ -185,14 +185,31 @@ private bool ApplyForceLinesOperation(AdjustNewLinesOperation operation, int pai
                 return true;
             }
 
+            private int getIndentation(int pairIndex, TriviaData triviaInfo)
```

Comment:
@basoundr I am not sure whether I am following you. I am basically saying change all those places where it requires specific behavior for its own purpose. that is what Rule is for.